parlai/core/params.py

```diff
@@ -333,6 +343,14 @@ def parse_args(self, args=None, namespace=None, print_args=True):
         if self.opt.get('datapath'):
             os.environ['PARLAI_DATAPATH'] = self.opt['datapath']
 
+        # nothing changes if model_file
```

Comment:
can you make this comment more clear?

Reply:
woops I think I was interrupted halfway through writing that, thanks for catching